lib/api-blueprint-preview-view.coffee

```diff
@@ -82,8 +82,9 @@ class ApiBlueprintPreviewView extends ScrollView
   renderApiBlueprintText: (text) ->
     fs.writeFileSync '/tmp/atom.apib', text
     # Env hack... helps find aglio binary
-    env =
-        PATH: process.env.PATH + ':/usr/local/bin'
+    env = Object.create(process.env)
+    npm_bin = atom.project.getPaths().map (p) -> "#{path.join(p, 'node_modules', '.bin')}:"
```

Comment:
What happens here when you have multiple projects open and each has its own local installation of Aglio? Won't we run into issues where the wrong one is selected? Presumably this isn't a big deal, but imagine having an old project open with an older version and you upgrade Aglio in your current project but the latest changes aren't available when previewed? I.e. it's using the older one by mistake. I'd like to prevent that kind of issue if possible.